src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -816,12 +816,12 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
                                           &(msg_params[strings::app_hmi_type]));
   }
 #endif  // SDL_REMOTE_CONTROL
+  SendResponse(true, result_code, add_info.c_str(), &response_params);
```

Comment:
@tkireva Please check that in RegisterAppInterfaceResponse there is no activities that should be done before OnAppRegistered.
Note comment in bellow.
@okozlovlux  please also participate in review